release-please-config.json

```diff
@@ -9,6 +9,7 @@
       "extra-files": [
         "README.md",
         "chart/Chart.yaml",
+        "chart/values.yaml",
```

Comment:
```suggestion
        "chart/ofo/values.yaml",
```

Reply:
```
❯ ls chart
Chart.yaml	templates	values.yaml
```

This is the content of the chart, it doesn't have an `ofo` sub folder

Reply:
I think the actual problem here is we might need `https://github.com/open-feature/open-feature-operator/pull/206/files#diff-cc5b1f6f50da569d760745a2cf39471cebd34940595c6c732e49cef8271a4030` (`charts/ofo/values.yaml`) added. It has a release-please annotation: https://github.com/open-feature/open-feature-operator/pull/206/files#diff-cc5b1f6f50da569d760745a2cf39471cebd34940595c6c732e49cef8271a4030R16

Reply:
Yeah, I was looking at this file.
https://github.com/open-feature/open-feature-operator/blob/bcc8c400ed8da97fa3aeb0a0ae8e8713581c7449/charts/ofo/values.yaml

Reply:
Okay that's user error - that chart shouldnt be there, @james-milligan and I were debugging. MY BAD!